### PR TITLE
add style guide

### PR DESCRIPTION
**Pull request by HAIL9000**

### CODE REVIEW

File: style_guide.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Style Guide (aka *✨the brand✨*)
+
+Because Django Girls PDX is all about creating a space where women and non-binary feel comfortable exploring programming, it's pretty important that we present ourselves in a way that helps build that atmosphere. Also, because we rely on many people and businesses in the community to keep make our workshop work, it's important to present ourselves in a way that's consistent and professional so people will continue to work with us.
```
Comment (spinecone) [inline style_guide.md]: non-binary [people/folks/something like that]

Comment (spinecone) [inline style_guide.md]: keep mak[ing] our workshop work

**Comment by HAIL9000** [comment]
@spinecone typos corrected!

**Comment by spinecone** [comment]
giphy 8

**Pull request merged**